From e98c49f2854ba5a17c2ef09fe6d2bb9bf8a9f4ed Mon Sep 17 00:00:00 2001
From: Shimin Zhou <shimingx.zhou@intel.com>
Date: Tue, 14 Jul 2015 17:01:00 +0800
Subject: [PATCH 412/429] usb: fix switching breakout board to host mode failed
 after reboot

Correct usb_event type, otherwise Edison breakout board
can not switch to host mode correctly after software
reboot.

Signed-off-by: Shimin Zhou <shimingx.zhou@intel.com>
---
 drivers/usb/dwc3/dwc3-intel-mrfl.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/dwc3/dwc3-intel-mrfl.c b/drivers/usb/dwc3/dwc3-intel-mrfl.c
index 981c84193ab3..ecd9b49a95c9 100644
--- a/drivers/usb/dwc3/dwc3-intel-mrfl.c
+++ b/drivers/usb/dwc3/dwc3-intel-mrfl.c
@@ -508,7 +508,7 @@ static int dwc3_intel_set_power(struct usb_phy *_otg,
 int dwc3_intel_enable_vbus(struct dwc_otg2 *otg, int enable)
 {
 	atomic_notifier_call_chain(&otg->usb2_phy.notifier,
-			USB_EVENT_DRIVE_VBUS, &enable);
+			USB_EVENT_VBUS, &enable);
 
 	return 0;
 }
-- 
2.37.3

